views/form.html

```diff
@@ -72,49 +62,44 @@ <h1 class="display-4">Onboarding Process form</h1>
-                            <span class="err">{{ @errors['ssn'] }}</span>
-                        </check>
-                        <div class="errors text-danger"></div>
-                    </div>
-                    <div class="form-group">
-                        <check if="{{ @bioCheck === 'on' }}">
+                    <div class="form-group val-empty">
+                        <label class="font-weight-bold">SSN</label><em>*</em>
+                        <check if='{{ @databaseSsn !=""}}'>
                             <true>
-                                <input class="checkBio" type="checkbox" name="bioCheck" checked="checked">
-                                <label><b>Update your bio</b></label><br>
-                                <div class="form-group not-hidden bio-block">
-                                    <label class="font-weight-bold">Bio</label><em>(optional)</em>
-                                    <textarea class="form-control input " name="bio"> {{ @bioText }}</textarea>
-                                    <check if="{{ @errors['check'] }}">
-                                        <span class="err">{{ @errors['check'] }}</span>
-                                    </check>
-                                </div>
+                                <span> <em>(We already have your SSN, update if there is any change</em>)</span>
+                                <input type="text" class="form-control input ssnInput" name="ssn"
+                                       value="{{ @ssn }}">
+                                <check if="{{ @errors['ssn'] }}">
+                                    <span class="err">{{ @errors['ssn'] }}</span>
+                                </check>
                             </true>
                             <false>
-                                <input class="checkBio" type="checkbox" name="bioCheck">
-                                <label>Update your bio</b></label><br>
-                                <div class="form-group hidden bio-block">
-                                    <label class="font-weight-bold">Bio</label><em>(optional)</em>
-                                    <textarea class="form-control input " name="bio"> {{ @bioText }}</textarea>
-                                </div>
+                                <input type="text" class="form-control input ssnInput" name="ssn"
+                                       value="{{ @ssn }}">
+                                <check if="{{ @errors['ssn'] }}">
+                                    <span class="err">{{ @errors['ssn'] }}</span>
+                                </check>
+                                <div class="errors text-danger"></div>
                             </false>
                         </check>
                     </div>
+                    <div class="form-group">
+                        <label class="font-weight-bold">Bio</label><em>(optional)</em>
```

Comment:
Could we maybe change the working from optional to something that indicates it eventually has to be filled out? Like: "Not yet submitted" or something along those lines, that would disappear once they have submitted a bio?

Reply:
Same for the picture upload.